### PR TITLE
1777 styles user settings dialog

### LINKED ISSUE #1777: Apply guidelines for user settings dialog
Please find Desktop and Phone Specification on this page
https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2104434679

### PR DESCRIPTION
**Pull request by UlianaMunich**
Fixes #1777

### CODE REVIEW

File: core/src/UserSettingsDialog.html [hunk 1]
```diff
@@ -208,15 +226,62 @@
...
 
+  let previousWindowWidth;
+  const onResize = () => {
+    const isMobileToDesktop =
+      window.innerWidth >= CSS_BREAKPOINTS.desktopMinWidth &&
+      previousWindowWidth < CSS_BREAKPOINTS.desktopMinWidth;
+    const isDesktopToMobile =
+      window.innerWidth < CSS_BREAKPOINTS.desktopMinWidth &&
+      previousWindowWidth >= CSS_BREAKPOINTS.desktopMinWidth;
+    console.log('resize');
```
Comment (legteodav) [inline core/src/UserSettingsDialog.html:260]:
```suggestion
```

**Review by JohannesDoberer — COMMENTED** [review]
Bildschirmfoto 2021-02-15 um 19 58 09
I guess it was not introduced in this pr but it would be nice to fix it here

**Comment by JohannesDoberer** [comment]
File
If I have a longer list on custom Mf I can't see the last entry on mobile view.

**Comment by JohannesDoberer** [comment]
Filemobile
If I have a longer list of entries in luigi editor I will not see the last element when I scroll to the end.

**Comment by JohannesDoberer** [comment]
Bildschirmfoto 2021-02-15 um 20 31 01
On standard view I would say the distance between content and footer is probably to high.

File: core/src/UserSettingsDialog.html [hunk 2]
```diff
@@ -379,114 +447,238 @@
...
-      height: 80vh;
-    }
-
     :global(.usersettings-leftNavVisible) .lui-usersettings-left-nav {
       width: 0;
     }
+    /*micro frontend and iframe wrappers inside the right-side dialog body*/
+    .mf-wrapper,
+    .iframe-wrapper {
+      overflow: auto;
```
Comment (JohannesDoberer) [inline core/src/UserSettingsDialog.html:477]: When I remove the `overflow:auto` I don't get a double scrollbar on iframe mf.
Currently I have it? Does it make sense?

File: core/src/UserSettingsDialog.html [hunk 3]
```diff
@@ -379,114 +447,238 @@
...
+    }
+
+    @media (max-width: 600px) {
+      :global(.usersettings-leftNavVisible) .lui-usersettings-left-nav {
+        width: 0;
+      }
+      /*micro frontend and iframe wrappers inside the right-side dialog body*/
+      .mf-wrapper,
+      .iframe-wrapper {
+        overflow: auto;
```
Comment (JohannesDoberer) [inline core/src/UserSettingsDialog.html:592]: I have a double scrollbar on iframe mf.
When I remove this `overflow:auto`, I don't have it.
Does it make sense? Can't see any side-effect.

**Review by legteodav — APPROVED** [review]

**Comment by JohannesDoberer** [comment]
IMG_7169
One small thing.. if I have a more entries in the nav I can't see the whole Item.

**Review by JohannesDoberer — CHANGES_REQUESTED** [review]
Please take a look to my suggestions. It fixes the scroll issue and the gap between last entry and bottom for all viewports. 
In addition the spec of fiori3 has bigger icons for the left nav.
Also please take a look to the font size and weight.

**Review by JohannesDoberer — APPROVED** [review]

**Pull request merged**